en/advanced_config/pid_tuning_guide_multicopter.md

```diff
@@ -1,217 +1,185 @@
 
-Parameters: [MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P).
 
-- Set `MC_YAWRATE_P` to small value, e.g. 0.1
+### Attitude Controller
 
-Keep the multi rotor in your hand and increase the thrust to about 50%,
-so that the weight is virtually zero. Turn it around its yaw axis,
-observe the response. The motor sound should change and the system
-should fight the yaw rotation. The response will be substantially weaker
-than roll and pitch, which is fine. If it oscillates or becomes twitchy,
-tune down `RATE_P`. If response is very large even on small movements
-(full throttle spinning vs idle spinning propellers) reduce `RATE_P`.
-Typical value is around 0.2…0.3.
+This controls the orientation and outputs desired body rates with the following tuning parameters:
+- Roll control ([MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P))
+- Pitch control ([MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P)
+- Yaw control ([MC_YAW_P](../advanced_config/parameter_reference.md#MC_YAW_P))
 
-The yaw rate control, if very strong or oscillating, can deteriorate the
-roll and pitch response. Check the total response by turning around,
-roll, pitch and yaw.
+The attitude controller is much easier to tune. 
+In fact, most of the time the defaults do not need to be changed at all.
 
+To tune the attitude controller, fly in *Manual/Stabilized mode* and increase the **P** gains gradually. 
+If you start to see oscillations or overshoots, the gains are too high.
 
-### Step 5: Stabilize Yaw Angle
+The following parameters can also be adjusted. These determine the maximum rotation rates around all three axes:
+- Maximum roll rate ([MC_ROLLRATE_MAX](../advanced_config/parameter_reference.md#MC_ROLLRATE_MAX))
+- Maximum pitch rate ([MC_PITCHRATE_MAX](../advanced_config/parameter_reference.md#MC_PITCHRATE_MAX)
+- Maximum yaw rate ([MC_YAWRATE_MAX](../advanced_config/parameter_reference.md#MC_YAWRATE_MAX))
 
 
-#### P Gain Tuning
+### Thrust Curve / Throttle PID Attenuation (TPA) {#thrust_curve}
 
-Parameters: [MC_YAW_P](../advanced_config/parameter_reference.md#MC_YAW_P).
+The tuning above optimises performance around the hover throttle. 
+But it can be that you start to see oscillations when going towards full throttle.
 
-- Set `MC_YAW_P` to a low value, e.g. 1
+There are two ways to counteract that:
+- Adjust the **thrust curve** with the [THR_MDL_FAC](../advanced_config/parameter_reference.md#THR_MDL_FAC) parameter (preferred method).
+  The thrust to PWM mapping is linear by default — setting `THR_MDL_FAC` to 1 makes it quadratic. 
+  Values in between use a linear interpolation of the two.
+  Typical values are between 0.3 and 0.5.
+  You can start off with 0.3 and then increase it by 0.1 at a time. 
+  If it is too high, you will start to notice oscillations at lower throttle values. 
+  
+  > **Note** The rate controller must be re-tuned if you change this parameter.

```

Comment:
Yes. The gains are not far off but still need readjustments, because a fixed difference in thrust will lead to different motor commands with a different factor.